python/ray/tune/trial.py

```diff
@@ -126,25 +130,30 @@ def stop(self, error=False):
                 stop_tasks.append(self.agent.stop.remote())
                 stop_tasks.append(self.agent.__ray_terminate__.remote(
                     self.agent._ray_actor_id.id()))
-                _, unfinished = ray.wait(
-                        stop_tasks, num_returns=2, timeout=10000)
-                if unfinished:
-                    print(("Stopping %s Actor was unsuccessful, "
-                           "but moving on...") % self)
+# TODO(ekl)  seems like wait hangs when killing actors
```

Comment:
This was put in place to address #1169. Commenting this out may introduce the issue again (despite it not showing up on immediate tests)?

Reply:
Does it actually return when you try it locally? What I observe is that it always sleeps 10s -- slowing down scheduling by orders of magnitude when running with fake data.

Reply:
Using wait, it returns 1 of them, but iirc returning simply 1 of them didn't help the tests. (Specific issue #1165)

Reply:
Then it should be ok to not wait at all, if it doesn't help the tests?

Reply:
Sorry, clarification - iirc simply returning 1 without the timer didn't fix the test issue, but putting in the timer resolved the issue. One reasonable alternative is just to decrease the timeout to 1 or 0.1 sec or or something.

Reply:
Seems reasonable, I changed the timeout to 250ms.